class ErrorMessage():
	#The error message for all caught exceptions.
	def errorMessage(self):
		print('\nSomething about that was not right. Try again. \n')